frontend/src/components/header/index.tsx

```diff
@@ -1,20 +1,17 @@
-import { IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/react'
-import { PropsWithChildren } from 'react'
+import { IonHeader, IonTitle, IonToolbar } from '@ionic/react'
 
 interface Props {
-  pageTitle: string
+  children: string
```

Comment:
Changing the type here?
(needs the import as well)

```suggestion
  children: React.ReactNode
```

Reply:
The Header should only render Text inside his child-context

This is allowed
```jsx
<Header>I am a beautiful text </Header>
```
This should not be allowed
```jsx
<Header> <BadComponent/> </Header>
```

Reply:
Then I'd call the prop title because children would imply that every react element can be used, right?

Reply:
Then it not working because the prop `title` is expected as prop like 
```jsx
<Header title={"Momente"}></Header> 
```